docs/src/zh/docs/formatter.md

```
@@ -0,0 +1,82 @@
+   "注意",
+   "djLint 并非 HTML 解析器或语法验证器。"
+%}
+
+## 示例
+
+### 处理前
+
+下面是一段急需处理的 HTML 代码块 -
+{% raw %}
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)